packages/compass-user-data/src/user-data.spec.ts

```diff
@@ -158,6 +158,20 @@ describe('user-data', function () {
         expect(mongoshData?.[1]).to.be.instanceOf(Stats);
       }
     });
+
+    it('reads many number of files', async function () {
+      const files = Array.from({ length: 10000 }, (_, i) => [
+        `data${i}.json`,
+        JSON.stringify({ name: `VSCode${i}` }),
+      ]);
+
+      await Promise.all(
+        files.map(([filepath, data]) => writeFileToStorage(filepath, data))
+      );
```

Comment:
Wouldn't this test also run the risk of running into file descriptor limitations?

Reply:
yeah that's a good point. i'll remove this test.